clang/lib/CodeGen/CGBuiltin.cpp

```diff
@@ -2313,14 +2313,20 @@ RValue CodeGenFunction::EmitBuiltinExpr(const GlobalDecl GD, unsigned BuiltinID,
       FD->hasAttr<AsmLabelAttr>() ? 0 : BuiltinID;
 
   bool ErrnoOverriden = false;
-  // True if math-errno is overriden via the
+  bool ErrnoOverrideValue = false;
```

Comment:
This might be easier to understand if you replaced these two variables with a single `std::optional<bool> ErrnoOverride`.